cmd/get.go

```diff
@@ -84,8 +84,14 @@ If no app is specified, the selected app (if configured) will be assumed.`,
 			log.Fatalf(color.RedString("Could not get provider type for provider '%s'"), provider)
 		}
 
+		// Set some sane default values for provider and app level durations
```

Comment:
Missing period.

This isn't the place to set Viper defaults - we are already inside the function at this stage. We probably want to do it in `init()`, unless you have a better idea.

Reply:
I've added the reasoning to the code. I'm not aware of setting a default for variable keys. I found that the nicer solution compared to trying to read and catching errors.